test/extended/util/annotate/rules.go

```diff
@@ -86,13 +86,21 @@ var (
 		// because of pullthrough not supporting ICSP (https://bugzilla.redhat.com/show_bug.cgi?id=1918376)
 		"[Skipped:Disconnected]": {
 			// Internet access required
+			`\[sig-apps\]\[Feature:DeploymentConfig\] deploymentconfigs with multiple image change triggers should run a successful deployment with a trigger used by different containers`,
+			`\[sig-apps\]\[Feature:DeploymentConfig\] deploymentconfigs with multiple image change triggers should run a successful deployment with multiple triggers`,
```

Comment:
I didn’t expect this - can you describe why this doesn’t work?  Triggers shouldn’t need the internet?

Reply:
Thanks for review @smarterclayton 

`[sig-apps][Feature:DeploymentConfig] deploymentconfigs with multiple image change triggers should run a successful deployment with multiple triggers` this test creates a deploymentconfig relies on `ruby:latest` and `mongodb:latest` whose are not existed in disconnected.  But is it expected this test should work even if the images are not in the list of `oc get istags`?, if yes, there seems to be a bug in somewhere else.

Reply:
After digging into these tests by running them on my disconnected local cluster;

These 2 tests create deploymentconfigs by using these fixtures https://github.com/openshift/origin/blob/master/test/extended/testdata/deployments/multi-ict-deployment.yaml, https://github.com/openshift/origin/blob/master/test/extended/testdata/deployments/deployment-simple.yaml

Deploymentconfigs can not reach healthy number even if the desired count is 1 and hence trigger step seems even not executed. Because there is no deploymentconfig yet.

I run `oc get istag -n openshift` and pick one of the already imported images(i.e. `tests:latest`), change the fixtures with this image, recompiled `openshift-tests` and run above tests in disconnected environment. Tests passed.

Reply:
So it seems like these two tests should go into their own new section, with a comment indicating that they fail because they pull images that we don't cache, and a link to a bz for that

Reply:
@danwinship, I moved these two tests in different section in `Skipped:Disconnected` list by adding a comment.

Reply:
No one should be using uncached images.  Looking at the referenced image, that's pointing to the local registry.  Why is the local registry unavailable?  It should be available and have the image referenced even if the cluster is disconnected.

Reply:
Local registry is available in disconnected but I think no one mirrors ruby and mongodb images. From what I understand we should fix these two tests by using already cached images and remove them from this PR.

Reply:
Oh, https://github.com/openshift/origin/blob/master/test/extended/testdata/deployments/deployment-simple.yaml is pointing to registry (tests should always be available), the other one is not (ruby) which causes it to fail.  Ok, then they're fine to be here.